### PR TITLE
#232 add hub logo favicon

### PR DESCRIPTION
**Pull request by annakukova**
#232 Changed favicon to neon green Hub logo with circular black background

### CODE REVIEW

File: packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import './awards_section.css';
 
 export const AwardsSection = () => {
     return (
-        <div className="awards-section-hack-aubg-container">
```
Comment (SuelaCark) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx:6]: Changes in the awards section shouldn't be here. You might have created a branch from the previous branch you have been working on. To avoid merge conflicts, please remove these changes made within the AwardsSection.jsx and awards_section.css files

File: packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css [hunk 2]
```diff
@@ -1,5 +1,4 @@
-.awards-section-hack-aubg-container {
-    margin-bottom: 5%;
+.awards-section-container {
```
Comment (SuelaCark) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css:1]: Same here^^

**Review by SuelaCark — CHANGES_REQUESTED** [review]
Please check the comments left in the file codes. Additionally, was the neon green suppose to be only for HackAUBG webpage and the other pages (landing page and jobs page) to have a neon blue color or smth similar that it will suit the HUB theme

File: packages/web/public/faviconcircle.ico:Zone.Identifier [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (asynchroza) [inline packages/web/public/faviconcircle.ico:Zone.Identifier:1]: Remove redundant files

File: packages/services/mailer_service/main.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+load_dotenv()
+
+
+def send_mail(receiver, subject, html):
+    port = 465  # SSL
+    email = environ.get('HUB_MAIL')
+    password = environ.get('HUB_PSW')
+
+    if not email or not password:
+        return json.dumps({"message": "Missing environment variables"}), 500
```
Comment (asynchroza) [inline packages/services/mailer_service/main.py:18]: Why is this here?

**Review by asynchroza — COMMENTED** [review]
Remove old favicons

**Comment by asynchroza** [comment]
@annakukova PRs are mixed up. I would suggest that you reset to master and build the PRs from the ground up. I see that the changes you've added are not that complex and easy to replicate so this shouldn't be a problem. Remember to always create your new branches from master. 

Cc: @SuelaCark

**Comment by SuelaCark** [comment]
@annakukova No need to worry about this PR or requested changes because I worked on them in this PR #299 
However, please work on having a separate neon blue HUB favicon for the landing page so we don't crash the matrix theme favicon to the landing page colors
